.gitignore

```diff
@@ -79,3 +79,6 @@ cagra_index
 ivf_flat_index
 ivf_pq_index
 
+# cuvs_bench
+datasets/
+/*.json
```

Comment:
Can we add back the trailing newline? And is there a name pattern that we can attach to this? Like “/bench*.json”? It’s very possible that we might want top level configuration in JSON.

Reply:
We'll rework this if we add top-level JSON configurations later on. For now, this tool sometimes create top-level JSON files so we'll refactor it, if needed, so it doesn't do that